Bottom: 904b40d2cb9910b261878c15181d4630fe476291
Top:    e038d032efcdae61996cc682614cb37225969576
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 17:18:04 -0500

Refresh of create-role

---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index 78ecf1a9..6e49eef7 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -1,6 +1,8 @@
 nexus_version: 3.13.0
 nexus_networks:
   - name: '{{ docker_network_frontend }}'
+    aliases:
+      - '{{ nexus_hostname }}'
 docker: true
 nexus_default_port: 80
 public_hostname: '{{ nexus_hostname }}'
